src/qrcode/QRCode.tsx

```diff
@@ -1,60 +1,65 @@
 import Clipboard from '@react-native-clipboard/clipboard'
-import React, { useState } from 'react'
+import React from 'react'
 import { Trans, useTranslation } from 'react-i18next'
 import { StyleSheet, Text, View } from 'react-native'
 import { useSelector } from 'react-redux'
 import { nameSelector } from 'src/account/selectors'
 import Button from 'src/components/Button'
-import ExchangesBottomSheet from 'src/components/ExchangesBottomSheet'
-import { ExternalExchangeProvider } from 'src/fiatExchanges/ExternalExchanges'
-import Paste from 'src/icons/Paste'
+import InLineNotification, { Severity } from 'src/components/InLineNotification'
+import CopyIcon from 'src/icons/CopyIcon'
 import StyledQRCode from 'src/qrcode/StyledQRCode'
 import { SVG } from 'src/send/actions'
+import { getDynamicConfigParams } from 'src/statsig'
+import { DynamicConfigs } from 'src/statsig/constants'
+import { StatsigDynamicConfigs } from 'src/statsig/types'
 import colors from 'src/styles/colors'
-import fontStyles from 'src/styles/fonts'
+import fontStyles, { typeScale } from 'src/styles/fonts'
 import { vibrateInformative } from 'src/styles/hapticFeedback'
 import variables from 'src/styles/variables'
+import { NetworkId } from 'src/transactions/types'
 import Logger from 'src/utils/Logger'
+import { networkIdToNetwork } from 'src/web3/networkConfig'
 import { walletAddressSelector } from 'src/web3/selectors'
 
 interface Props {
   qrSvgRef: React.MutableRefObject<SVG>
-  exchanges: ExternalExchangeProvider[]
-  onCloseBottomSheet?: () => void
   onPressCopy?: () => void
-  onPressInfo?: () => void
-  onPressExchange?: (exchange: ExternalExchangeProvider) => void
 }
 
 export default function QRCodeDisplay(props: Props) {
   const { t } = useTranslation()
-  const { exchanges, qrSvgRef } = props
+  const { qrSvgRef } = props
   const address = useSelector(walletAddressSelector)
   const displayName = useSelector(nameSelector)
 
-  const [bottomSheetVisible, setBottomSheetVisible] = useState(false)
-
-  const onCloseBottomSheet = () => {
-    props.onCloseBottomSheet?.()
-    setBottomSheetVisible(false)
-  }
-
   const onPressCopy = () => {
     props.onPressCopy?.()
     Clipboard.setString(address || '')
     Logger.showMessage(t('addressCopied'))
     vibrateInformative()
   }
 
-  const onPressInfo = () => {
-    props.onPressInfo?.()
-    setBottomSheetVisible(true)
-  }
-
-  const onPressExchange = (exchange: ExternalExchangeProvider) => {
-    props.onPressExchange?.(exchange)
+  const getSupportedNetworks = () => {
+    const supportedNetworkIds = getDynamicConfigParams(
+      DynamicConfigs[StatsigDynamicConfigs.MULTI_CHAIN_FEATURES]
+    ).showBalances
+    const networks = supportedNetworkIds.map((id: NetworkId) => {
+      const network = networkIdToNetwork[id]
+      return network.charAt(0).toUpperCase() + network.slice(1)
+    })
+    return networks.join(', ')
   }
 
+  const description = () => (
+    <Text style={styles.description}>
```

Comment:
is the text wrapper needed here? seems like the text style is duplicated here and in the InLineNotification component